activemodel/lib/active_model/validations/acceptance.rb

```diff
@@ -20,6 +20,11 @@ def setup!(klass)
         klass.send(:attr_reader, *attr_readers)
         klass.send(:attr_writer, *attr_writers)
       end
+
+      def acceptable_option?(value)
+        accept_options = [options[:accept]].flatten
```

Comment:
Works for me. I'll update shortly. :)
On Jan 10, 2015 5:07 PM, "Sean Griffin" <notifications@github.com> wrote:

> In activemodel/lib/active_model/validations/acceptance.rb
> <https://github.com/rails/rails/pull/18439#discussion-diff-22763094>:
>
> > @@ -20,6 +20,11 @@ def setup!(klass)
> >          klass.send(:attr_reader, *attr_readers)
> >          klass.send(:attr_writer, *attr_writers)
> >        end
> > +
> > +      def acceptable_option?(value)
> > +        accept_options = [options[:accept]].flatten
>
> How about we just do Array(options[:accept]) instead?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/rails/pull/18439/files#r22763094>.
>